docs/source/rbac/roles.md

```diff
@@ -92,7 +92,7 @@ c.JupyterHub.load_roles = [
 ]
 ```
 
-The role `reader` allows users `maria` and `joe` and service `external` to read (but not modify) any user’s model.
+The role `reader` allow users `maria` and `joe` and service `external` to read (but not modify) any users model.
```

Comment:
These changes are both not correct: "The role allows users to ... any user's model"